area/models.py

```diff
@@ -52,8 +52,8 @@ class AreaName(ComplexField):
     object_ref = models.ForeignKey('Area')
     value = models.TextField(default=None, blank=True, null=True)
     field_name = _("Area of Operations")
-    shortcode = 'u_an'
-    spreadsheet_field_name = 'unit:area_ops_name'
+    # shortcode = 'u_ln'
```

Comment:
TODO: Should I remove or update these?

Reply:
Yes, I think we should update the shortcodes. I'll prepare a list for you.